server/routes/model/reportSummary.js

```diff
@@ -41,7 +41,8 @@ const createUseOfForceDetails = (details = {}) => {
 const createRelocation = (relocationAndInjuries = {}) => {
   return {
     prisonerRelocation: toLabel(RelocationLocation, relocationAndInjuries.prisonerRelocation),
-    prisonerCompliancy: relocationAndInjuries.relocationCompliancy,
+    prisonerCompliant: relocationAndInjuries.prisonerCompliant,
```

Comment:
I don't think you can just change this? Won't this break for existing reports? 
Got to think of the data stored in json payloads in the DB.

Reply:
you are quite right

Reply:
agreed